galaxy.yml

```diff
@@ -1,7 +1,7 @@
 namespace: vmware
 name: vmware_rest
 readme: README.md
-version: null
+version: 2.3.2-dev0
```

Comment:
Shouldn't it be 2.3.1?

Reply:
We have to decide what version to add to the collections. I see *-dev0 version added to amazon.aws. @jillr @gravesm your thoughts, please?

Reply:
I set it to the correct version in [community.vmware](https://github.com/ansible-collections/community.vmware) when I release and set it back to `null` afterwards: [History for community.vmware/galaxy.yml](https://github.com/ansible-collections/community.vmware/commits/main/galaxy.yml)

Don't know if this helps.

ansible-collections/community.vmware#1320